owncloudApp/build.gradle

```diff
@@ -53,6 +53,8 @@ dependencies {
     //Zooming Android ImageView.
     implementation 'com.github.chrisbanes:PhotoView:2.1.4'
 
+    implementation 'com.android.support:multidex:1.0.3'
```

Comment:
Would it be possible to use androidx one?

Reply:
Yes, it makes sense